app/controllers/service_requests_controller.rb

```diff
@@ -185,8 +185,7 @@ def obtain_research_pricing
     else
       to_notify = update_service_request_status(@service_request, 'get_a_cost_estimate')
     end
-
-    send_confirmation_notifications(to_notify)
```

Comment:
Is it possible for NotifierLogic to figure out to_notify in send_confirmation_notifications? It should probably be in there.